### PR TITLE
ref(test): Convert Project CSP Report to RTL + TS - [TET-360]

### PR DESCRIPTION
**Pull request by priscilawebdev**

### CODE REVIEW

File: static/app/views/settings/projectSecurityHeaders/projectCspReports.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    );
+
+    const mock = MockApiClient.addMockResponse({
+      url: projectUrl,
+      method: 'PUT',
+    });
+
+    expect(mock).not.toHaveBeenCalled();
+
+    // Click Regenerate Token
```
Comment (evanpurkhiser) [inline static/app/views/settings/projectSecurityHeaders/projectCspReports.spec.tsx:100]: Looks like this is typing not clicking

Comment (priscilawebdev) [reply static/app/views/settings/projectSecurityHeaders/projectCspReports.spec.tsx:100]: oh yes.. good catch... I might have only copied/paste

Comment (priscilawebdev) [inline static/app/views/settings/projectSecurityHeaders/projectCspReports.spec.tsx:100]:
```suggestion
```

Comment (priscilawebdev) [reply static/app/views/settings/projectSecurityHeaders/projectCspReports.spec.tsx]: no no...it was there before and I think it makes sense to delete it

Comment (evanpurkhiser) [reply static/app/views/settings/projectSecurityHeaders/projectCspReports.spec.tsx]: whoops lol

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**